setuptools/tests/test_bdist_wheel.py

```diff
@@ -311,7 +331,7 @@ def test_licenses_deprecated(dummy_dist, monkeypatch, tmp_path):
     bdist_wheel_cmd(bdist_dir=str(tmp_path)).run()
 
     with ZipFile("dist/dummy_dist-1.0-py3-none-any.whl") as wf:
-        license_files = {"dummy_dist-1.0.dist-info/DUMMYFILE"}
+        license_files = {"dummy_dist-1.0.dist-info/licenses/licenses/DUMMYFILE"}
```

Comment:
May be worth to add a comment with a note for future maintainers along the lines of:

> The file structure already has a `licenses` folder. PEP 639 adds an extra `licenses` folder inside the metadata directory.

Or change the example to use a different directory name all together when creating the test project directory (clearer approach, but more work).